src/pages/Rockets.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchRocketsData,
+  reserveRocket,
+  cancelReserveRocket,
+} from '../redux/rockets/rocketsSlice';
+import './styles/Rockets.css';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const {
+    isLoading, data, error, joinedRockets,
+  } = useSelector(
+    (state) => state.rockets,
+  );
+  useEffect(() => {
+    dispatch(fetchRocketsData());
+  }, [dispatch]);
+  const handleReserveButtonClick = (rocketId) => {
+    dispatch(reserveRocket(rocketId));
+  };
+  const handleCancelReservationButtonClick = (rocketId) => {
+    dispatch(cancelReserveRocket(rocketId));
+  };
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+  if (error) {
+    return <div>{error}</div>;
+  }
+  return (
+    <div className="rockets-container">
+      {data.map((rocket) => (joinedRockets.includes(rocket.id) ? (
+        <div className="rocket-left" key={rocket.id}>
+          <img
+            src={rocket.image}
+            alt={rocket.name}
+            className="rocket-image"
+          />
+          <div className="rocket-details">
+            <h2 className="rocket-name">{rocket.name}</h2>
+            <p className="rocket-desc">
+              {' '}
+              <span className="reserved-badge">Reserved</span>
+              {rocket.description}
+            </p>
+            <button
+              type="button"
+              className="rocket-btn"
+              data-testid={`cancel-reservation-button-${rocket.id}`}
+              onClick={() => handleCancelReservationButtonClick(rocket.id)}
+            >
+              Cancel Reservation
+            </button>
```

Comment:
- [ ] @Team great work displaying rockets, however, the design of the Cancel Reservation  button doesn't match the design given, check the comparison below

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/meryemsanem/react-capstone/assets/61408860/9c7ae4fb-3281-4598-af20-75e77504c5d0) |  ![image](https://github.com/meryemsanem/react-capstone/assets/61408860/75c88861-0bc1-48d8-b32b-3cea3e109cc6)